flags.go

```diff
@@ -57,6 +57,10 @@ func defineFlagVar() *FlagVar {
 		"Enabling KCP Watcher to reconcile Watcher CRs created by KCP run operators")
 	flag.StringVar(&flagVar.skrWatcherPath, "skr-watcher-path", "./skr-webhook",
 		"The path to the skr watcher chart.")
+	flag.BoolVar(&flagVar.enableTLSCertSync, "enable-tls-cert-sync", true,
+		"Enabling synchronisation of TLS-Certificate-Secrets, created during the Watcher installation")
+	flag.StringVar(&flagVar.tlsCertSecretName, "tls-cert-secret-name", "skr-webhook-tls",
```

Comment:
```suggestion
	flag.StringVar(&flagVar.tlsCertSyncSecretName, "tls-cert-sync-secret-name", "skr-webhook-tls",
```

Reply:
This is anyway a leftover from old commits - I will remove it completely